<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addClassDialog</class>
 <widget class="QDialog" name="addClassDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add class</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="commentLineEdit">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>29</y>
     <width>95</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="commentLabel">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>29</y>
     <width>149</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment(not necessarily):</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>37</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>1</y>
     <width>95</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="patternMatchRoleComboBox">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>80</y>
     <width>74</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="patternMatchRoleLabel">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>80</y>
     <width>106</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pattern Match Role:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="roleComboBox">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>51</y>
     <width>74</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="roleLabel">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>51</y>
     <width>29</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Role:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addClassDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addClassDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
